src/main/resources/languages/language.ttl

```diff
@@ -55,8 +55,8 @@ ufo:object-type a a-popis-dat:term ;
 
 ufo:relator a a-popis-dat:term ;
     rdfs:label "Vztah"@cs,
-        "Relator"@en .
+        "Relation"@en .
 
 ufo:relator-type a a-popis-dat:term ;
     rdfs:label "Typ vztahu"@cs,
-        "Relator Type"@en .
+        "Relation Type"@en .
```

Comment:
Relator

Reply:
So it's "Relation" but "Relator type"?

Reply:
No, I meant Relator, not 'Relator type'. 

But to be sure I looked again in [doc](https://dev.nemo.inf.ufes.br/seon/UFO.html) and I was wrong, it is vice versa, so:

Relation Type -> Relator
Relator Type -> Relation

Reply:
I still don't get it. The language (as can be seen in the file) contains only "Relator" and "Relator type". So how do we change their translation?